app/config/FrontendAppConfig.scala

```diff
@@ -181,6 +179,14 @@ class FrontendAppConfig @Inject()(val servicesConfig: ServicesConfig, val config
 
   val encryptionIsEnabled: Boolean = config.get[Boolean]("encryption.isEnabled")
 
-  lazy val readFeatureSwitchesFromMongo = servicesConfig.getBoolean("feature-switches.read-from-mongo")
+  lazy val readFeatureSwitchesFromMongo: Boolean = servicesConfig.getBoolean("feature-switches.read-from-mongo")
+
+  lazy val isTimeMachineEnabled: Boolean = servicesConfig.getBoolean("time-machine.enable-time-machine")
+
+  lazy val isTimeMachineAddYearEnabled: Boolean = servicesConfig.getBoolean("time-machine.add-year.enable-time-machine-add-year")
```

Comment:
I don't think we need to have so many config entries: for days, months and years; should be fine to have delta for day ie

Reply:
I think a delta would be more confusing. To go from today to Jan 31st you would have to work out the Delta which is annoying to do. Plus, if you wanted to end up on the same specific date when testing over a few days, you would have to change the config once it hits midnight